docs/CHANGELOG.asciidoc

```diff
@@ -221,10 +222,55 @@ Machine Learning::
 Kibana platform::
 * Fixes an issue with the global search field that could open the wrong page when pressing "Enter" while results were not yet fully loaded ({kibana-pull}197750[#197750]).
 
+[[release-notes-8.16.2]]
+== {kib} 8.16.2
+
+The 8.16.2 release includes the following enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.16.2]]
+=== Enhancements
+In this release, we've introduced an image based on the hardened link:https://github.com/wolfi-dev/[Wolfi] image to provide secure containers to our self-managed customers, help with compliance regulations, and improve our supply chain security posture. Wolfi-based images require Docker version 20.10.10 or higher.
```

Comment:
```suggestion
In this release, we've introduced an image based on the hardened link:https://github.com/wolfi-dev/[Wolfi] image to provide additional security to our self-managed customers, and improve our supply chain security posture. Wolfi-based images require Docker version 20.10.10 or higher.
```

Minor rewording per internal discussion :)

Reply:
RIght, I forgot about that! Thanks @leemthompo 😄